internal/k3d/terraform.go

```diff
@@ -53,3 +59,62 @@ type GithubTerraformEnvs struct {
 	AwsAccessKeyId        string
 	AwsSecretAccessKey    string
 }
+
+// TerraformPrep prepares Terraform files by detokenizing them.
+// It processes files in the specified GitOps directory and the GitHub runner path.
+func TerraformPrep(config *K3dConfig) error {
+	// Define paths for Terraform and GitHub runner components
+	terraformPath := filepath.Join(config.GitopsDir, "terraform")
+	githubRunnerPath := filepath.Join(config.GitopsDir, "registry/kubefirst/")
+
+	log.Info().Msgf("Repo is %s", terraformPath)
+
+	// Detokenize Terraform files
+	if err := filepath.Walk(terraformPath, detokenizeTerraform(terraformPath, config)); err != nil {
+		return fmt.Errorf("error in detokenizing Terraform: %w", err)
+	}
+
+	log.Info().Msg("Applying to GitHub runner")
+
+	// Detokenize GitHub runner files
+	if err := filepath.Walk(githubRunnerPath, detokenizeTerraform(githubRunnerPath, config)); err != nil {
+		return fmt.Errorf("error in detokenizing GitHub runner: %w", err)
+	}
+	log.Info().Msgf(githubRunnerPath)
```

Comment:
This log statement calls `Msgf` but doesn't have any formatting directive. It also seems to print randomly the runner path but nothing else. Perhaps it needs more context?